android_build command fails

# Bug

`downloadDependencies` task not found when running `android_build` command

```

FAILURE: Build failed with an exception.

* What went wrong:
Task 'downloadDependencies' not found in root project 'XXXX'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3m 8s
Exited with code 1
```
## Environment info

<!--
  Run `react-native info` in your terminal and copy the results here. Also, include the *precise* version number of this library that you are using in the project
-->

React native info output:

```bash
React Native Environment Info:
    System:
      OS: macOS 10.14.5
      CPU: (8) x64 Intel(R) Core(TM) i7-7920HQ CPU @ 3.10GHz
      Memory: 1.26 GB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
      Yarn: 1.17.3 - ~/.nvm/versions/node/v10.16.0/bin/yarn
      npm: 6.9.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
      Android SDK:
        API Levels: 23, 24, 25, 26, 27, 28, 29
        Build Tools: 23.0.1, 23.0.2, 23.0.3, 24.0.1, 25.0.0, 26.0.1, 26.0.2, 26.0.3, 27.0.0, 27.0.3, 28.0.2, 28.0.3, 29.0.0, 29.0.1
        System Images: android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5692245
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      react: ^16.8.3 => 16.9.0
      react-native: 0.59.4 => 0.59.4
    npmGlobalPackages:
      react-native-cli: 2.0.1
```
## Steps To Reproduce

On a fresh react-native project

1. `android/gradlew downloadDependencies -p android` will not find the task to download dependencies

This is the same as the step [here](https://github.com/react-native-community/react-native-circleci-orb/blob/master/src/commands/android_build.yml#L46)

Describe what you expected to happen:

1. I guess because this task doesn't exist that it shouldn't be called. Ideally there's another task or some way to add the task to the project

--- a/README.md
+++ b/README.md
@@ -23,7 +23,27 @@ You will also need to ensure that you have a MacOS plan enabled if you want to b
 ## Documentation
 [You can read the full documentation here](https://circleci.com/orbs/registry/orb/react-native-community/react-native).
 
+## Android
+
+Add this task in your <example>/app/build.gradle
+
+```gradle
+task downloadDependencies() {
+  description 'Download all dependencies to the Gradle cache'
+  doLast {
+    configurations.findAll().each { config ->
+      if (config.name.contains("minReactNative") && config.canBeResolved) {
+        print config.name
+        print '\n'
+        config.files
+      }
+    }
+  }
+}
+```
+
 ## Example
+
 Here is a full example of how the Orb can be used in a CircleCI workflow to build and test a React Native app:
 
 ```yaml

